<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAddLive</class>
 <widget class="QDialog" name="DlgAddLive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgAddLive</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>直播类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combLiveType">
       <item>
        <property name="text">
         <string>RTMP 直播</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RDLiveSDK 云直播</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>锐动直播 (云直播)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>RTMP 直播地址：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="texRtmpUrl">
         <property name="maxLength">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>例如：rtmp://192.168.0.1/live/abc</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>用户ID（能标识用户唯一性的字符串）：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="texSdkUserId">
         <property name="maxLength">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>小写字母和数字的组合，小于32个字符</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="texRdLiveUserName">
         <property name="maxLength">
          <number>64</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="texRdLivePassword">
         <property name="maxLength">
          <number>64</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="chkRdLiveAutoLogin">
         <property name="text">
          <string>自动登录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="butOK">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>combLiveType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
